### ISSUE TITLE
[ACCESSIBILITY] The Status Card mode not available - Showstopper

### ISSUE THREAD

**Issue by timo54** [rokwire/safer-illinois-app#158]
**Describe the bug**
On the Status Card window, there are two views - one of the Building Access status, the other is the QR code to identify the user. These two views can be accessed with a two-finger horizontal swipe while using the screen reader but they are not announced. The two-buttons which illustrate which window you're on are not focusable via swiping and are not announced. The screen reader user will not know these two screens are available and will not know how to switch between them to show their QR code identity or their building access status.

**To Reproduce**
1. Turn the screen reader on
2. Go to the Status Card
3. Status Card mode buttons cannot be focused upon

**Expected behavior**
Every element on the screen should receive focus as one swipes through the screen while using screen reading software.

**Screenshots**
See below.

**Smartphone (please complete the following information):**
- Device: Google Pixel
- OS: Android
- Version 2.5.8

<img width="418" alt="UnreachableButtons" src="https://user-images.githubusercontent.com/12497650/93362856-227af180-f80c-11ea-86c6-4f17fb5bc90a.png">

**Comment by mdryankov** [comment]
@timo54  I have added Page1 & Page 2 labels + put actions to the page indicators (e.g Page 1 of 2, Page 2 of 2...). 
Please let me know if this is acceptable from your point of view, because it was not a standard situation.

**Comment by timo54** [comment]
@mdryankov, I only have version 2.5.8 on Android and iOS to test and in both cases, this issue still exists. Please advise on which version I need to test. Thanks.

**Comment by mdryankov** [comment]
@timo54, Sorry for the delay. You could use the app store version 2.6.13 (Android, I guess iOS too). It contains the fix for sure.
